plugins/node/opentelemetry-plugin-ioredis/src/utils.ts

```diff
@@ -84,35 +84,40 @@ export const traceSendCommand = (
     this: ioredisTypes.Redis & IORedisPluginClientTypes,
     cmd?: IORedisCommand
   ) {
-    if (arguments.length >= 1 && typeof cmd === 'object') {
-      const span = tracer.startSpan(cmd.name, {
-        kind: SpanKind.CLIENT,
-        attributes: {
-          [AttributeNames.COMPONENT]: IORedisPlugin.COMPONENT,
-          [AttributeNames.DB_TYPE]: IORedisPlugin.DB_TYPE,
-          [AttributeNames.DB_STATEMENT]: dbStatementSerializer(
-            cmd.name,
-            cmd.args
-          ),
-        },
-      });
+    if (arguments.length < 1 || typeof cmd !== 'object') {
```

Comment:
```suggestion
    if (arguments.length < 1 || typeof cmd !== 'object') {
```

When does/could this happen? can we put a comment above e.g.

Reply:
Since we monkey patch the runtime we code we can never be sure how the module is used, those check are present in most plugins to prevent us from instrumenting a usage that we didn't anticipated.
I don't think its really worth to explain that since it's inherent of the monkeypatch method

Reply:
Ok thanks for explaining, so it's to protect from unanticipated use of the lib, but we may not have a concrete example (hence unanticipated).